pkg/barrier/barrier.go

```diff
@@ -225,6 +227,7 @@ func NewFrameworkBarrier() *FrameworkBarrier {
 
 func (b *FrameworkBarrier) Run() {
 	klog.Infof("Running %v", ComponentName)
+	ctx := context.TODO()
```

Comment:
>context.TODO() [](http://example.com/codeflow?start=8&length=14)

Why previously use the context.Background, and now changed to context.TODO? Are they both work? What is the difference?